roop/enhancer.py

```diff
@@ -180,8 +183,8 @@ def multi_process_frame(source_img, frame_paths, progress) -> None:
 
 def process_video(source_path: str, frame_paths: list[str], mode: str) -> None:
     progress_bar_format = '{l_bar}{bar}| {n_fmt}/{total_fmt} [{elapsed}<{remaining}, {rate_fmt}{postfix}]'
-    total = len(frame_paths)
-    with tqdm(total=total, desc='Processing', unit='frame', dynamic_ncols=True, bar_format=progress_bar_format) as progress:
+    total = len(frame_paths) + state.processed_frames_cnt
```

Comment:
the state has to provide a method like `state.count_unfinished_frames()`

Reply:
I tried to avoid the usage of global variables here. But it is not really matters.